services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -642,6 +642,68 @@ class FlowCrudSpec extends BaseSpecification {
         Wrappers.wait(WAIT_OFFSET) { assert !islUtils.getIslInfo(newIsl).isPresent() }
     }
 
+    def "Systems allows to pass traffic via full and vlan port when they are on the same port"() {
+        when: "Create a flow with vlan port"
+        def (Switch srcSwitch, Switch dstSwitch) = topology.activeTraffGens*.switchConnected
+        def bandwidth = 100
+        def flowVlanPort = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        flowVlanPort.maximumBandwidth = bandwidth
+        flowVlanPort.allocateProtectedPath = true
+        flowHelper.addFlow(flowVlanPort)
+
+        and: "Create the same flow with full port"
+        def flowFullPort = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        flowFullPort.maximumBandwidth = bandwidth
+        flowFullPort.source.vlanId = 0
+        flowFullPort.destination.vlanId = 0
+        flowFullPort.allocateProtectedPath = true
+        flowHelper.addFlow(flowFullPort)
```

Comment:
You should use the same ports on src and dst. Current test is not relevant

Reply:
You may be mistaken.
Here we create flows with traffgen ports.
```
FlowCreatePayload randomFlow(Switch srcSwitch, Switch dstSwitch, boolean useTraffgenPorts = true,
                                 List<FlowPayload> existingFlows = []) {
```
As a result this test alsways gets the same port for both flows

Reply:
agree